### PR TITLE
updated build instructions to match current process

### PR DESCRIPTION
**Pull request by DavidjohnBlodgett**
@RackHD/corecommitters 

Just cleaning up some instructions that were not changed when the on-imagebuilding process was updated to handle later versions of ansible.

### CODE REVIEW

File: docs/rackhd/microkernel.rst [hunk 1]
```diff
@@ -64,39 +64,72 @@ with the base image built by the existing ansible roles in this repository.
 Building Images
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To build images, define an imagebuilding playbook (see example.yml for an example) or
-use the default one (all.yml)::
+To build images, define an imagebuilding script (see example.sh for an example) or
+use the default one (all.sh)::
```
Comment (anhou) [inline docs/rackhd/microkernel.rst]: 'all.sh' should be 'build_all.sh'

**Comment by anhou** [comment]
the same contents are maintained in two places https://github.com/RackHD/on-imagebuilder/blob/master/README.md. so maybe it's better to add a link to https://github.com/RackHD/on-imagebuilder/blob/master/README.md ? @RackHD/corecommitters 
@heckj

File: docs/rackhd/microkernel.rst [hunk 2]
```diff
@@ -64,39 +64,72 @@ with the base image built by the existing ansible roles in this repository.
...
+
+    ```
+    sudo ansible-playbook -i hosts common/initrd_wrapper.yml \
+    -e "config_file=vars/initrd.yml provisioner=roles/initrd/provision_initrd"
+
+    sudo ansible-playbook -i hosts common/basefs_wrapper.yml \
+    -e "config_file=vars/basefs.yml provisioner=roles/basefs/provision_rootfs"
+    ```
+
+  so it should run `build_all.sh` which include inirtd and basefs playbook before `build_oem.sh`
```
Comment (pscharla) [inline docs/rackhd/microkernel.rst]: Just a small typo to fix the spelling of "initrd".  Also, not sure if this is the right place, but did you want/need any description of commands to run the "build_oem.sh" script?

**Comment by DavidjohnBlodgett** [comment]
@anhou I totally agree, with everyones blessings... I'd like to just link to the readme, since I'm sure that will be updated more consistently.

**Comment by DavidjohnBlodgett** [comment]
@pscharla @benbp So I'm a little hazy on my memory of our exact expectation with the oem code... I feel like I remember a conversation that implied that it was just there as a reference but cannot work without licensed dependences... Ben if I'm incorrect and we should document using this, I'll be glad to make it happen.

**Comment by pscharla** [comment]
@DavidjohnBlodgett In regards to the oem code, I wouldn't worry about it for now (if this is the only thing blocking the PR).  Go ahead and get the changes in and then we can always add the OEM instructions later if needed.

**Comment by DavidjohnBlodgett** [comment]
@heckj do you have any thoughts on the question @anhou brought up? I feel like there was an early requirement we maintain these docs in parallel to our readme docs. I would assume that it's a slippery slope adding a link to the readme, since at the end of the day..."in my opinion" always going to the readme is the best (and most understood) place to get info about various aspects of this tool. (so I side with Andrew here)

**Comment by DavidjohnBlodgett** [comment]
@RackHD/corecommitters ok, implemented @anhou 's suggestions for links to our readme. I think this is merge ready.

**Comment by DavidjohnBlodgett** [comment]
@RackHD/corecommitters hold that thought... I did markdown syntax... need to correct for rst.

**Comment by tldavies** [comment]
@DavidjohnBlodgett Squash the commits

**Comment by DavidjohnBlodgett** [comment]
@tldavies squashed!

**Comment by benbp** [comment]
👍

**Pull request merged**